common/random.go

```diff
@@ -2,30 +2,45 @@ package common
 
 import (
 	crand "crypto/rand"
-	"math/rand"
+	mrand "math/rand"
+	"sync"
 	"time"
 )
 
 const (
 	strChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" // 62 characters
 )
 
-func init() {
+// pseudo random number generator.
+// seeded with OS randomness (crand)
```

Comment:
`mrand`, not `crand` ??

Reply:
Nope, it is seeded with crand, see init()